python/Ganga/new_tests/GPI/GangaUnitTest.py

```diff
@@ -169,3 +182,10 @@ def tearDown(self):
         stop_ganga()
         import sys
         sys.stdout.flush()
+
+    @classmethod
+    def tearDownClass(cls):
+        if cls.wipe_repo is True:
+            import shutil
+            shutil.rmtree(cls.gangadir, ignore_errors=True)
```

Comment:
hi all, came to convo late so may not have all the facts but why not set it in a setUpClass classmethod?

```python

@classmethod
def setUpClass(cls):
    cls.gangadir = <blah>
```